scripts/build.tokens.ckerc20.mjs

```diff
@@ -24,9 +24,10 @@ const orchestratorInfo = async ({ orchestratorId: canisterId }) => {
 const buildOrchestratorInfo = async (orchestratorId) => {
 	const { managed_canisters } = await orchestratorInfo({ orchestratorId });
 
-	const mapManagedCanisters = (acc, { ledger, index, ckerc20_token_symbol }) => {
+	const mapManagedCanisters = (acc, { ledger, index, ckerc20_token_symbol, erc20_contract }) => {
 		const ledgerCanister = fromNullable(ledger);
 		const indexCanister = fromNullable(index);
+		const erc20ContractAddress = erc20_contract.address;
```

Comment:
wanna get fancy and spare this line? you can destruct the `address` when the param is destructed.

```
const mapManagedCanisters = (acc, { ledger, index, ckerc20_token_symbol, erc20_contract: {address} }) => {
```

Reply:
yes!!!